modules/exploits/unix/webapp/trixbox_ce_auth_rce.rb

```diff
@@ -9,9 +9,9 @@ class MetasploitModule < Msf::Exploit::Remote
 
   def initialize(info = {})
     super(update_info(info,
-      'Name'            => 'Trixbox CE <= v2.8.0.4 Authenticated Remote Command Execution',
+      'Name'            => 'Trixbox CE v2.8.0.4 Authenticated Remote Command Execution',
       'Description'     => %q{
-        This module exploits a post-authentication OS command injection vulnerability found in Trixbox CE <= v2.8.0.4 which may allow arbitrary command execution on the underlying operating system.
+        This module exploits a post-authentication OS command injection vulnerability found in Trixbox CE v2.8.0.4 which may allow arbitrary command execution on the underlying operating system.
```

Comment:
If this affects older versions and you have confirmed this, please revert this change. Only the title of the exploit module needed to be changed.

Reply:
This was later confirmed to affect older versions so this was reverted back to what it was before by including the <=.